src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,70 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+
+    // ADD THE PORT NUMBERS TO CONSTANTS!!!!!
+
+    public static final class DriveConstants {
+        public static final int kDriveMotor1 = 0;
+        public static final int kDriveMotor2 = 0;
+        public static final int kDriveMotor3 = 0;
+        public static final int kDriveMotor4 = 0;
+
+        public static final int kLeftTalonPort = 0;
+        public static final int kRightTalonPort = 0;
+        
+        public static final int kGyroSensor = 0; //IMU for point turn
+
+        public static final double kWheelDiameter = 0; //ADD UNIT OF MEASUREMENT
+
+    }
+
+    public static final class LauncherConstants {
+        public static final int kLauncherMotor1 = 0;
+        public static final int kLauncherMotor2 = 0;
+
+        // pistons adjust the angle of the launcher
+        public static final int kLauncherPiston1 = 0; // double solenoid
```

Comment:
There's only one solenoid, but it's driving two pistons.  Change these to kLauncherSolenoid or kLauncherPistons.